src/Aspire.Dashboard/Components/Pages/StructuredLogs.razor.css

```diff
@@ -54,3 +54,7 @@
     grid-area: foot;
     padding-bottom: calc(var(--design-unit) * 2px);
 }
+
+::deep .wrap {
+    text-wrap: wrap;
```

Comment:
This wraps content. But it doesn't respect any of the whitespace (line breaks, indentation) in the content.

For example, this exception message is just one long wrapped line without applying any of the content's whitespace:

![image](https://github.com/dotnet/aspire/assets/303201/2b478ffc-a49f-4f0b-a228-4e541bcf5774)

Reply:
responded [here](https://github.com/dotnet/aspire/pull/2087#discussion_r1480422514)